enacts/crop_suitability/layout_crop_suit.py

```diff
@@ -41,6 +41,7 @@ def app_layout():
     lon_max = np.around((rr_mrg["X"][-1]+lon_res/2).values, decimals=10)
     lat_label = str(lat_min)+" to "+str(lat_max)+" by "+str(lat_res)+"˚"
     lon_label = str(lon_min)+" to "+str(lon_max)+" by "+str(lon_res)+"˚"
+    target_year = str(rr_mrg["T"][-1].dt.year.values)
```

Comment:
Are you sure you've tested this? `values` returns an array, so I would have thought `str(x.values)` came out something like `[1995]` instead of `1995`. But if you're sure it works then it's fine.

Reply:
Also (sorry for nitpicking) I think `max_year` would be a better name for this variable. If we decide to use something other than the most recent year as the default target, this value will no longer be the target but it will still be the max.

Reply:
tested again and also printed it to see it and it's just the 4 characters of the year...

no problem with nitpicking, half of them tend to have crossed my mind and I was too lazy to act on it.